pypinfo/core.py

```diff
@@ -88,12 +88,32 @@ def parse_query_result(query):
     return rows
 
 
+def add_percentages(rows):
+
+    headers = rows.pop(0)
+    index = headers.index('download_count')
+    headers.insert(index, 'percent')
+    total_downloads = sum(int(row[index]) for row in rows)
+
+    for r, row in enumerate(rows):
+        percent = '{:.1%}'.format(int(row[index]) / total_downloads)
```

Comment:
Can you please remove the `%` when output is json? Alternatively, you could just get rid of it.

Reply:
Another option would be to display it as the fractional contribution (75% ==> 0.75).

This would make `--percent` a less natural name for the argument, though.